odoo_module_migrate/migration.py

```diff
@@ -155,12 +158,18 @@ def _get_migration_scripts(self):
             if script_start >= migration_end or script_end <= migration_start:
                 continue
 
-            self._migration_scripts.append(importlib.import_module(full_name))
+            module = importlib.import_module(full_name)
+            migrationScript = getattr(module, "MigrationScript")()
+            migrationScript.name = module.\
+                __file__.\
+                split('/')[-1].\
+                split('.')[-2]
```

Comment:
These 6 lines are a perfect candidate for a `load_migration_script(full_name)` method, as they are repeated in https://github.com/OCA/odoo-module-migrator/pull/28/files#diff-3ef061fbb7d7092e9cbaea551f09d8d01896c397b31927350dd1c608844c9e4bR126-R129